simplified-ui-splash/src/main/java/org/nypl/simplified/ui/splash/MailtoWebViewClient.java

```diff
@@ -0,0 +1,52 @@
+package org.nypl.simplified.ui.splash;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.MailTo;
+import android.net.Uri;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import java.lang.ref.WeakReference;
+
+/**
+ * Used to handle mailto: links in the eula.html
+ */
+public class MailtoWebViewClient extends WebViewClient {
+    private final WeakReference<Activity> mActivityRef;
+
+    public MailtoWebViewClient(Activity activity) {
+        mActivityRef = new WeakReference<>(activity);
+    }
+
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        if (url.startsWith(MailTo.MAILTO_SCHEME)) {
+            final Activity activity = mActivityRef.get();
+            if (activity != null) {
+                try {
```

Comment:
You can drop the `try / catch` here since you're calling `resolveActivity ` instead.